requirements.txt

```diff
@@ -1,2 +1,2 @@
 openapi-spec-validator
```

Comment:
Can we lock the version and enable dependabot for future upgrades?

Reply:
Yes....will add that in. Good idea